frontend/src/components/SubmissionWidget.tsx

```diff
@@ -114,15 +111,25 @@ const SubmissionWidget: React.FC<SubmissionWidgetProps> = ({
               </h4>
               {globalDocs.map((doc) => (
                 <div key={doc.title} className="mb-4">
-                  <ul>
-                    <Link
-                      to={generateDocRoute(doc.title, true)}
-                      target="_blank" // This is what is causing the issue.
-                      rel="noopener noreferrer"
-                      className="underline text-blue-600 hover:text-blue-800"
-                    >
-                      {doc.title}
-                    </Link>
+                  <ul className="list-disc pl-6 text-black">
+                    <li>
+                      <button
+                        onClick={() => openDocInNewTab(doc)}
+                        className="text-blue-500 hover:text-blue-300"
+                      >
+                        {doc.title}
+                      </button>
+                      <li>
+                        <Link
+                          to="../../public/python-3.13-docs-html/index.html"
+                          target="__blank"
+                          rel="noopener noreferrer"
+                          className="text-blue-500 hover:text-blue-300"
+                        >
+                          Python 3 Documentation
```

Comment:
I moved the link outside of the docs.map in order to get it to show up. Upon doing this, the python documentation did finally show up. The problem with the current system is that when you go through the python documentation, the console log gets filled with the below warnings. This will not be good in deployment or development, as it may obscure any meaningful app interaction. 

```
11:40:13 frontend.1 | Files in the public directory are served at the root path.
11:40:13 frontend.1 | Instead of /public/python-3.13-docs-html/index.html, use /python-3.13-docs-html/index.html.
11:40:13 frontend.1 | Files in the public directory are served at the root path.
11:40:13 frontend.1 | Instead of /public/python-3.13-docs-html/_static/pygments.css?v=80d5e7a1, use /python-3.13-docs-html/_static/pygments.css?v=80d5e7a1.
11:40:13 frontend.1 | Files in the public directory are served at the root path.
11:40:13 frontend.1 | Instead of /public/python-3.13-docs-html/_static/pydoctheme.css?v=41b4f12d, use /python-3.13-docs-html/_static/pydoctheme.css?v=41b4f12d.
11:40:13 frontend.1 | Files in the public directory are served at the root path.
11:40:13 frontend.1 | Instead of /public/python-3.13-docs-html/_static/sidebar-wrap.css?v=65806c5a, use /python-3.13-docs-html/_static/sidebar-wrap.css?v=65806c5a.
11:40:13 frontend.1 | Files in the public directory are served at the root path.
11:40:13 frontend.1 | Instead of /public/python-3.13-docs-html/_static/pygments_dark.css?v=b20cc3f5, use /python-3.13-docs-html/_static/pygments_dark.css?v=b20cc3f5.
11:40:13 frontend.1 | Files in the public directory are served at the root path.
11:40:13 frontend.1 | Instead of /public/python-3.13-docs-html/_static/documentation_options.js?v=6aaf4047, use /python-3.13-docs-html/_static/documentation_options.js?v=6aaf4047.
11:40:13 frontend.1 | Files in the public directory are served at the root path.
11:40:13 frontend.1 | Instead of /public/python-3.13-docs-html/_static/doctools.js?v=9bcbadda, use /python-3.13-docs-html/_static/doctools.js?v=9bcbadda.
11:40:13 frontend.1 | Files in the public directory are served at the root path.
11:40:13 frontend.1 | Instead of /public/python-3.13-docs-html/_static/sphinx_highlight.js?v=dc90522c, use /python-3.13-docs-html/_static/sphinx_highlight.js?v=dc90522c.
11:40:13 frontend.1 | Files in the public directory are served at the root path.
11:40:13 frontend.1 | Instead of /public/python-3.13-docs-html/_static/sidebar.js, use /python-3.13-docs-html/_static/sidebar.js.
11:40:13 frontend.1 | Files in the public directory are served at the root path.
11:40:13 frontend.1 | Instead of /public/python-3.13-docs-html/_static/pydoctheme_dark.css, use /python-3.13-docs-html/_static/pydoctheme_dark.css.
11:40:13 frontend.1 | Files in the public directory are served at the root path.
11:40:13 frontend.1 | Instead of /public/python-3.13-docs-html/_static/copybutton.js, use /python-3.13-docs-html/_static/copybutton.js.
11:40:13 frontend.1 | Files in the public directory are served at the root path.
11:40:13 frontend.1 | Instead of /public/python-3.13-docs-html/_static/menu.js, use /python-3.13-docs-html/_static/menu.js.
11:40:13 frontend.1 | Files in the public directory are served at the root path.
11:40:13 frontend.1 | Instead of /public/python-3.13-docs-html/_static/search-focus.js, use /python-3.13-docs-html/_static/search-focus.js.
11:40:13 frontend.1 | Files in the public directory are served at the root path.
11:40:13 frontend.1 | Instead of /public/python-3.13-docs-html/_static/themetoggle.js, use /python-3.13-docs-html/_static/themetoggle.js.
11:40:13 frontend.1 | Files in the public directory are served at the root path.
11:40:13 frontend.1 | Instead of /public/python-3.13-docs-html/_static/rtd_switcher.js, use /python-3.13-docs-html/_static/rtd_switcher.js.
11:40:13 frontend.1 | Files in the public directory are served at the root path.
11:40:13 frontend.1 | Instead of /public/python-3.13-docs-html/_static/py.svg, use /python-3.13-docs-html/_static/py.svg.
11:40:13 frontend.1 | Files in the public directory are served at the root path.
11:40:13 frontend.1 | Instead of /public/python-3.13-docs-html/_static/py.svg, use /python-3.13-docs-html/_static/py.svg.
11:40:13 frontend.1 | Files in the public directory are served at the root path.
11:40:13 frontend.1 | Instead of /public/python-3.13-docs-html/_static/opensearch.xml, use /python-3.13-docs-html/_static/opensearch.xml.
```

Reply:
I saw this too just now and fixed it. Because public is part of the root of the project, I just changed it up to be and that seemed to have gotten rid of the messages you see in the terminal. Here is the modified code:

```jsx
<Link
      to="/python-3.13-docs-html/index.html"
      target="__blank"
      rel="noopener noreferrer"
      className="text-blue-500 hover:text-blue-300"
    >
      Python 3 Documentation
    </Link>
```